example/lib/main.dart

```diff
@@ -1,10 +1,20 @@
-import 'package:flutter/material.dart';
 import 'dart:async';
 
-import 'package:flutter/services.dart';
 import 'package:customer_io/customer_io.dart';
+import 'package:customer_io/customer_io_config.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+
+void main() async {
+  WidgetsFlutterBinding.ensureInitialized();
+
+  await CustomerIo.initialize(
```

Comment:
Assuming that `await` in Dart is performing asynchronous operations, does `initialize()` need to be asynchronous since it's not in the native SDKs?

Reply:
```suggestion
  await CustomerIO.initialize(
```